ui/job-view/pushes/JobButton.jsx

```diff
@@ -78,6 +78,8 @@ export default class JobButtonComponent extends React.Component {
     const resultStatus = state === 'completed' ? result : state;
     const runnable = state === 'runnable';
     const btnClass = getBtnClass(resultStatus, failure_classification_id);
+    // TODO: This is broken (bug 1504713)
+    // eslint-disable-next-line no-restricted-globals
     let title = `${resultStatus} | ${job_type_name} - ${status}`;
```

Comment:
I think we can take `` - ${status}`` out of the template here.  I can't think of what it was supposed to be at this point, since ``resultStatus`` covers both ``result`` and ``state``.

Reply:
Yeah I agree - I've left a comment on bug 1504713 :-)
(Won't fix here so as to not introduce behaviour changes in this PR)